packages/assets-controllers/src/TokenBalancesController.ts

```diff
@@ -206,10 +206,10 @@ export class TokenBalancesController extends BaseController<
         newContractBalances[address] = toHex(
           await this.#getERC20BalanceOf(address, selectedAddress),
         );
-        token.balanceError = null;
+        token.hasBalanceError = false;
```

Comment:
Just a thought, changing this property name will imply a migration to the client application

Reply:
@tommasini exactly a migration would be required for this change